_author_instructions/01_policies.md

```diff
@@ -257,21 +246,15 @@ Documents should not have been submitted in the current form to another journal,
 Preprints do not count as prior publication. 
 Documents that are major revisions of previously published articles are welcomed. 
 However, authors should ensure that any material they publish in LiveCoMS is not subject to licensing restrictions (such as from another journal) which impedes its release under the selected license.
-Some journals, for example, *Annual Reviews* lets the authors retain the right to create derivative works, which could perhaps be exercised in preparing a review to be published in LiveCoMS. 
+Some journals, e.g. *Annual Reviews*, lets the authors retain the right to create derivative works, which could perhaps be exercised in preparing a review to be published in LiveCoMS. 
 
-If an article is an adaptation of a previously published article, it must be noted in the cover letter, and major changes noted. 
+If an article is an adaptation of a previously published article, it must be noted in the submission cover letter and major changes noted. 
 Evaluating whether such changes constitute a significant revision will be part of the review process.
 
 Authors should, to the extent possible, determine the author order among themselves.  
-However, we acknowledge that institutions
```

Comment:
Did you think this bit was not worth mentioning?

Reply:
(Will merge to get the larger issues cleaned up and we can deal with this in a separate PR if needed).

Reply:
Ah, I see. This is another instance of duplication. Thanks for fixing.